mcs/tools/resx2sr/resx2sr.cs

```diff
@@ -136,7 +155,18 @@ static void GenerateFile (List<Tuple<string, string, string>> txtStrings, CmdOpt
 
 				str.WriteLine ();
 			}
+
```

Comment:
This change is wrong and broke the tool - and it's not just the cosmetic issue of not adding newlines here, which makes the output look _extremely ugly_.